src/contexts/StateContextProvider.tsx

```diff
@@ -85,6 +86,7 @@ const STATE_DEFAULT: Readonly<StateContextType> = Object.freeze({
     numPages: 0,
     onDiskFileSizeInBytes: 0,
     pageNum: 0,
+    queryProgress: 0,
```

Comment:
Following https://github.com/y-scope/yscope-log-viewer/pull/107/files#r1837630385 , shall we use the const here?
```suggestion
    queryProgress: INITIAL_QUERY_PROGRESS,
```

Reply:
We might want to rename this const as `QUERY_PROGRESS_VALUE_MIN`. see https://github.com/y-scope/yscope-log-viewer/pull/107/files#r1837632878